source/usr/lib/mcvirt/parser.py

```diff
@@ -365,6 +365,29 @@ def __init__(self, verbose=True):
         self.network_subparser.add_parser('list', help='List the networks on the node',
                                           parents=[self.parent_parser])
 
+        #Create subparser for VNC commands
+        self.vnc_parser = self.subparsers.add_parser('vnc', help='Connect or disconnect from VNC',
+                                                    parents=[self.parent_parser])
+        self.vnc_subparser = self.vnc_parser.add_subparsers(
+            dest='vnc_action',
+            metavar='Action',
+            help='Action to perform for the VNC'
+        )
+        self.vnc_connect_parser = self.vnc_subparser.add_parser(
+            'connect',
+            help='Connect to the VNC server',
+            parents=[self.parent_parser]
+        )
+        self.vnc_connect_parser.add_argument('vm_name', metavar='VM Name', type=str,
```

Comment:
Are you able to add this to the vnc_parser, after specifying the subparsers to add the 'VM Name' to both subparsers?